### ISSUE TITLE
[Bug]: Validation Error on subscribe_trade_updates

### ISSUE THREAD

**Issue by yohay13** [alpacahq/alpaca-py#210]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

when subscribing to trade updates stream with `trade_stream.subscribe_trade_updates(on_new_data_callback)`
I sometimes get this following error, after which the stream stops working:
```
error during websocket communication: 1 validation error for TradeUpdate
execution_id
  field required (type=value_error.missing)
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/alpaca/trading/stream.py", line 172, in _run_forever
    await self._consume()
  File "/usr/local/lib/python3.8/site-packages/alpaca/trading/stream.py", line 145, in _consume
    await self._dispatch(msg)
  File "/usr/local/lib/python3.8/site-packages/alpaca/trading/stream.py", line 89, in _dispatch
    await self._trade_updates_handler(self._cast(msg))
  File "/usr/local/lib/python3.8/site-packages/alpaca/trading/stream.py", line 103, in _cast
    result = TradeUpdate(**msg.get("data"))
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for TradeUpdate
execution_id
  field required (type=value_error.missing)
```

### Expected Behavior

work

### SDK Version I encountered this issue in

0.6.1

### Steps To Reproduce

```markdown
1. subscribe to the trades update stream
2. do some trades
3. get error
```

### Filled out the Steps to Reproduce section?

- [X] I have entered valid steps to reproduce my issue or have attached a minimally reproducible case in code that shows my issue happening; and understand that without this my issue will be flagged as invalid and closed after 30 days.

**Comment by forLinDre** [comment]
I am getting this error from the very first trade update, however was not experiencing the same issue at the time @yohay13 
 created it. In fact, everything was running smoothly up until yesterday around noon for me. @OlegRa is the issue you mentioned that was implemented for the csharp api going to be implemented for the Python version? I am getting the error from the very first trade update, every single time.

**Comment by forLinDre** [comment]
I was able to resolve this by changing my local Alpaca-py TradeUpdate class contained within trading/models.py as follows:

class TradeUpdate(BaseModel):
    event: Union[TradeEvent, str]
    # execution_id: UUID
    **execution_id: Optional[UUID]**
    order: Order
    timestamp: datetime
    position_qty: Optional[float]
    price: Optional[float]
    qty: Optional[float]

**Comment by haxdds** [comment]
Closed by #234
